### PR TITLE
Load messages branch

### PR DESCRIPTION
**Pull request by GavinAshworth**
Implemented message loading upon channel enter. Still need to add username functionality but that will happen when preston implements his work

### CODE REVIEW

File: PythonScripts/sendMessage.py [hunk 1]
```diff
@@ -25,3 +33,14 @@ def send_message(text, username, time, curr_channel):
...
     app.db.update_channel(curr_channel, channel_information1)
     return jsonify({'success': True})  # Return a success response
+
+def load_messages(channel):
+    # Fetch channels from the database
+    selected_channel = app.db.retrieve_from_channel(channel)
+    if selected_channel is None:
+        return []
+    messages = selected_channel["messages"]
+    print(messages)
```
Comment (imoudu13) [inline PythonScripts/sendMessage.py:43]: remove print statement

File: PythonScripts/sendMessage.py [hunk 2]
```diff
@@ -25,3 +33,14 @@ def send_message(text, username, time, curr_channel):
...
+    if messages is None:
```
Comment (imoudu13) [inline PythonScripts/sendMessage.py:44]: Remove this if block, there will never be a case when the messages is None

**Review by imoudu13 — CHANGES_REQUESTED** [review]
You forgot to test your code as well.

**Review by imoudu13 — CHANGES_REQUESTED** [review]

File: PythonScripts/sendMessageTest.py [hunk 3]
```diff
@@ -38,6 +38,29 @@ def test_send_message_failure(self, mock_db):
             # Check if the function returns a failure response
             self.assertNotEqual(response.json, {'success': False})
 
+    @patch('sendMessage.app.db')
+    def test_load_messages_with_existing_channel(self, mock_db):
+        # Mock the database handler
+        mock_channel_info = {"messages": ["Message 1", "Message 2"]}
```
[the remark this answers is not in the document]
Comment (GavinAshworth) [reply PythonScripts/sendMessageTest.py:44]: It just makes it so we dont have to actually get it from the database. Makes the tests run alot faster

**Pull request merged**